packages/lambdas/src/sqs-to-converter.ts

```diff
@@ -198,10 +198,22 @@ export const handler = Sentry.AWSLambda.wrapHandler(async (event: SQSEvent) => {
           timestamp: new Date(),
         };
 
+        const preProcessedFilename = `${s3FileName}.from_converter`;
```

Comment:
`.from_converter.json`?

Reply:
Yea this is to differentiate between the one after post processing

Reply:
oh yea need to add json